themes/theme-ios11/widgets/ProductSlider/index.jsx

```diff
@@ -100,6 +102,7 @@ class ProductSlider extends Component {
   render() {
     const { settings } = this.props;
     const { sliderSettings } = settings;
+    const { slidesPerView = 2.3 } = useWidgetSettings(WIDGET_ID) || {};
```

Comment:
React does not support hooks in class components. Please change to usage of the HOC instead.

Reply:
HOC: `withWidgetSettings`
Access to widget settings via: `const { slidesPerView = 2.3 } = this.props.widgetSettings || {};`